app/templating/summary/answer.py

```diff
@@ -1,7 +1,7 @@
 class Answer:
 
-    def __init__(self, answer_schema, answer, child_answer_value=None):
-        self.id = answer_schema['id']
+    def __init__(self, answer_schema, answer, group_instance, child_answer_value=None):
+        self.id = answer_schema['id'] + '-' + str(group_instance)
```

Comment:
Maybe a silly question, but can group instance ever be None or ''?  If so, are we okay with an id of `answer_id-`

Reply:
I can't see an obvious path for `group_instance` to be None. If it is, then I think we're going to have issues elsewhere as well.

I double checked this by checking that group_instance was never set to None in the debugger while running most of the unit / integration tests.